transport/http/src/test/scripts/org/kaazing/gateway/transport/http/test.http.handshake.timeout.rpt

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+connect tcp://localhost:8001
+connected
+
```

Comment:
It is unclear to me whether the http layer will kick in right away here.  I would suggest putting the read await BARRIER lower then the first line "GET /clientaccess" line.  And then after that I think there is no need to write out the rest of the headers, as it will race the close from the server